test/pseudo-tty/ref_and_unref.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+require('../common');
+
+const TTY = process.binding('tty_wrap').TTY;
+
+const handle = new TTY(0);
+handle.readStart();
+
+handle.unref();
+handle.ref();
+
+var timeout = setTimeout(function() {
```

Comment:
Can we not use a timer. They tend to make tests flakey. Is there anything in `process._getActiveHandles()` that can be checked here instead?